### ISSUE TITLE
[BUG]Server Crash on Startup with build 163

### ISSUE THREAD

**Issue by DMoate** [zmaster587/AdvancedRocketry#1662]
#1341  Version of Advanced Rocketry
>build 163

## Have you verified this is an issue in the latest unstable build
- Yes

## Version of LibVulpes
>LibVulpes 1.12.2-0.4.1-1-UNSTABLE build -1 for 1.12.2
Released 8/5/19

## Version of Minecraft
>1.12.2

## Does this occur without other mods installed
- No

## Crash report or log or visualVM (if applicable)
http://pastebin.com is a good place to put them
crash reports that are put in the issue itself are
hard to read
>https://pastebin.com/vhFjSdr2

## Description of the problem
>After the latest update to Advanced Rocketry our server crashes on start up, tried multiple times but same thing kept happening. Tried without any other mods and it seems to load fine but the log said it was caused by Advanced Rocketry so I have no idea where the problem is happening.

**Comment by dmodoomsirius** [comment]
Instead of the log file. Post the trash report inside the crash report folder please.

**Comment by DMoate** [comment]
Sorry my bad here ya go:
[crash-2019-08-12_12.00.06-server.txt](https://github.com/zmaster587/AdvancedRocketry/files/3490928/crash-2019-08-12_12.00.06-server.txt)

**Comment by dmodoomsirius** [comment]
Try the full pack without MoCreatures. Slowly remove mods until it works. As something is hanging the watchdog thread.

**Comment by DMoate** [comment]
So with 140+ mods I decided to try something else rather than arbitrarily removing each mod one at a time to see what the cause is and I instead deleted all config files for all mods then tried again. Seems to be working now with fresh new configs. If I had to guess though it may have been the new update with "Bedrock B Gone" mod as that is now server side only and is constantly spamming the server window as it replaces all bedrock in the world. Either way it looks to be fixed, thank you for your time!
